perllib/FixMyStreet/App/Form/Claims.pm

```diff
@@ -478,12 +478,22 @@ has_field what_cause => (
     required => 1,
     label => 'What was the cause of the incident?',
     options => [
-        { label => 'Bollard', value => 'bollard' },
-        { label => 'Cats Eyes', value => 'catseyes' },
-        { label => 'Debris', value => 'debris' },
+        { label => 'Bollard', value => 'bollard', data_hide => '#form-what_cause_other-row' },
+        { label => 'Cats Eyes', value => 'catseyes', data_hide => '#form-what_cause_other-row' },
+        { label => 'Debris', value => 'debris', data_hide => '#form-what_cause_other-row' },
+        { label => 'Other', value => 'other', data_show => '#form-what_cause_other-row' },
```

Comment:
FWIW, I'm not sure this list is comprehensive, I only pulled enough in from the prototype to test but I don't know where Martin got the full list in the prototype.

Reply:
Ah, right, thanks, have checked against the prototype and updated the missing options here and on where cause, and fixed a few missing hints too